LongWarOfTheChosen/Localization/LW_Overhaul.rus

```diff
@@ -612,3 +612,23 @@ m_strNextHaven="СЛЕД. УБЕЖИЩЕ"
 [Logistics X2LWAlienActivityTemplate]
 ActivityObjectives[0]="Захватить набор ресурсов \"Адвента\""
 +MissionDescriptions=(MissionFamily="SmashnGrab_LW", Description="Сопротивление обнаружило расположение ценных ресурсов противника которые мы можем заполучить для своих потребностей.", MissionIndex=0)
+
+; LWOTC Translated
+ActivityObjectives[0]="Начало освобождения региона"
+UpToText="До"
+ExpectedActivityTextStr="Ожидаемая активность"
+BoostedActivityText="Ожидаемая активность (с ускорением)"
+BoostTextPre="С"
+BoostTextPost="Ускорение"
+; End Translated
```

Comment:
This block should have a `[UISquadSelect_InfiltrationPanel]` section header, i.e.
```
[UISquadSelect_InfiltrationPanel]
UpToText="До"
...
```
In fact, it should be moved to that section, since the header already exists in this file.

Reply:
done